server/lib/adapters/postgres.js

```diff
@@ -118,7 +121,6 @@ export function runQuery(client, query, options = {}) {
       // Run the query.
       currentQuery = client.query(query, (queryError, result) => {
         if (timer) clearTimeout(timer);
-        client.end();
```

Comment:
I checked, we were calling client.end() twice every time we run query. For postgres connector it was fine but mysql connector was throwing an error that we try end the connection that was already closed. I removed this client.end() only for runQuery methods and in sync-agent we are always invoking `this.adapter.in.closeConnection()` so we are always closing it